iron-overlay-behavior.html

```diff
@@ -115,7 +111,10 @@
       closingReason: {
         // was a getter before, but needs to be a property so other
         // behaviors can override this.
-        type: Object
+        type: Object,
+        value: {
```

Comment:
This previously wasn't set because `canceled` had a default value and its observer would update the `closingReason`.
If I let the default value of `canceled` to false, then this should be reverted.